localenv/cloud-nine-wallet/docker-compose.yml

```diff
@@ -70,8 +70,9 @@ services:
       EXCHANGE_RATES_URL: http://cloud-nine-wallet/rates
       REDIS_URL: redis://shared-redis:6379/0
       WALLET_ADDRESS_URL: ${CLOUD_NINE_WALLET_ADDRESS_URL:-https://cloud-nine-wallet-backend/.well-known/pay}
-      ILP_CONNECTOR_ADDRESS: ${CLOUD_NINE_CONNECTOR_URL}
```

Comment:
Some thing are hardcoded here in compose file, some in .env
Maybe we should decide how do we want to handle this, just for the sake of consistency